.coveragerc

```diff
@@ -8,7 +8,7 @@ omit =
         .tox/*
         *.pyenv*
         *.venv*
-
+        */AppData/*
```

Comment:
On Github Action WIndows instances, the Python install lives in `AppData`, so we need to ignore that directory when calculating the coverage results.